CHANGES.rst

```diff
@@ -32,6 +32,14 @@ New Features
 
 - ``astropy.nddata``
 
+  - ``NDData.__init__``: got an additional optional parameter ``copy``
+    which is False by default. If it is True all other parameters are copied
+    before saving them as attributes. If False the parameters are only copied
+    if there is no way of saving them as reference.
+
+  - ``NDData.__init__``: ``Masked_Quantity`` objects are allowed as ``data``
```

Comment:
+ NDData.__init__ -> NDData
+ Add the issue number in brackets at the end: `[#4270]`